src/opencensus.erl

```diff
@@ -185,8 +185,7 @@ put_attributes(NewAttributes, Span=#span{attributes=Attributes}) ->
 %%--------------------------------------------------------------------
 -spec generate_trace_id() -> non_neg_integer().
 generate_trace_id() ->
-    <<Id:128>> = crypto:strong_rand_bytes(16),
-    Id.
+    uniform(340282366920938463463374607431768211456).
```

Comment:
either `uniform(2 bsl 127). % 2 shifted left by 127 == 2 ^ 128` or at least `%  = 2 ^ 128` would make this more understandable why we pick a crazy number. Plus less likely to be wrong, i.e. if you'd use `340282366920938463563374607431768211456` there no one would have ever noticed. (same for the `18446744073709551616` below)